Library/Homebrew/cask/cask.rb

```diff
@@ -24,6 +24,10 @@ class Cask
     attr_accessor :download, :allow_reassignment
 
     def self.all
+      # if !ARGV.include?("--all") && !Homebrew.EnvConfig.trust_all?
+      #   odeprecated "Cask::Cask#all without --all or HOMEBREW_TRUST_ALL"
+      # end
+
```

Comment:
Why is this commented out?

Reply:
Adding a comment but: so it can be uncommented for 3.7.0